components/core/tests/test-EncodedVariableInterpreter.cpp

```diff
@@ -376,6 +376,43 @@ TEST_CASE("EncodedVariableInterpreter", "[EncodedVariableInterpreter]") {
         ));
     }
 
+    SECTION("Test multiple metching values") {
+        char const cVarDictPath[] = "var.dict";
+        char const cVarSegmentIndexPath[] = "var.segindex";
+        vector<string> var_strs = {"python2.7.3", "Python2.7.3", "PyThOn2.7.3", "PYTHON2.7.3"};
+        clp::VariableDictionaryWriter var_dict_writer;
```

Comment:
Nit: let's add some new lines so it's easier to read.
```suggestion

        clp::VariableDictionaryWriter var_dict_writer;
```